policyengine_us/variables/gov/states/ca/tax/income/credits/earned_income/ca_eitc.py

```diff
@@ -89,6 +89,17 @@ def modify_parameters(parameters: ParameterNode) -> ParameterNode:
         return simulation.branches.get("ca_eitc")
 
 
+class ca_eitc_eligible(Variable):
```

Comment:
Could you move this to `ca_eitc_eligible.py` and import the branch from here?

Alternatively we could start making branch-making functions their own files, like `_get_ca_eitc_branch.py`. @nikhilwoodruff wdyt?

Reply:
@MaxGhenis said:
> Could you move this to `ca_eitc_eligible.py` and import the branch from here?

What is the syntax for "import the branch from here"?

Reply:
@MaxGhenis,  If you aren't sure how to import a branch into another module, could you merge PR #1897 as it is now.  Then you and @nikhilwoodruff can experiment with putting the `ca_eitc_eligible` variable in its own module.  Let me point out that having two or more variables defined in the same module has been done in PolicyEngineUS from the very beginning.   So, this PR is not out of line in this respect with the existing code base.

Reply:
Sounds good. We would like to move all variables into their own folder (#373). I also filed https://github.com/PolicyEngine/policyengine-us/issues/1906 to use `defined_for` here after testing. But merging for now.